api/views.py

```diff
@@ -416,3 +418,30 @@ class Accounts(generics.RetrieveAPIView):
     def get(self, request):
         serializer = AccountSerializer(request.user)
         return response.Response(serializer.data, status=status.HTTP_200_OK)
+
+
+class ContactUs(generics.CreateAPIView):
+    serializer_class = ContactUsSerializer
```

Comment:
Since the ContactUs page does not require the user to be logged in, we should add the AllowAny permission
```suggestion
class ContactUs(generics.CreateAPIView):
    serializer_class = ContactUsSerializer
    permission_clsses = (permissions.AllowAny)
```

Reply:
```suggestion
class ContactUs(generics.CreateAPIView):
    serializer_class = ContactUsSerializer
    permission_classes = (permissions.AllowAny,)
```
There are some mistakes in that suggestion that wouldn't have the correct effect if committed directly

Reply:
What will happen using this suggestion?

Reply:
The mistake appears to have already been committed. `permission_clsses` is misspelt. It should be `permission_classes` like in my revised suggestion. And the value should be a tuple. The comma is important after the `permissions.AllowAny` in the parentheses.